include/uapi/sound/sof/tokens.h

```diff
@@ -198,4 +198,9 @@
 /* COPIER */
 #define SOF_TKN_INTEL_COPIER_NODE_TYPE		1980
 
+/* ACP I2S */
+#define SOF_TKN_AMD_ACPI2S_RATE			1700
+#define SOF_TKN_AMD_ACPI2S_CH			1701
+#define SOF_TKN_AMD_ACPI2S_TDM_MODE		1702
```

Comment:
do you have a SOF PR for this change as well?

Reply:
This two tokens already merged in sof firmware long back.
#define SOF_TKN_AMD_ACPI2S_RATE			1700
#define SOF_TKN_AMD_ACPI2S_CH			1701


#define SOF_TKN_AMD_ACPI2S_TDM_MODE		1702 --->This token will be added in latest sof firmware patches..